src/middlewares/notification.middleware.js

```diff
@@ -0,0 +1,19 @@
+import PreferenceService from '../services/preference.service';
+import ResponseService from '../services/response.service';
+
+const doesOptionExist = async (req, res, next) => {
```

Comment:
`doesOptionExist` is not a good function name. Ensure your function name has a verb and clearly describes what the function does.